90zfsbootmenu/module-setup.sh

```diff
@@ -162,6 +162,25 @@ install() {
   inst_hook initqueue/settled 99 "${moddir}/hook/zfsbootmenu-ready-set.sh" || _ret=$?
   inst_hook initqueue/finished 99 "${moddir}/hook/zfsbootmenu-ready-chk.sh" || _ret=$?
 
+  # If tracing is enabled, build in the full-weight profiling library
+  if [ -n "${zfsbootmenu_enable_tracing}" ]; then
+    inst_simple "${moddir}/profiling/profiling-lib.sh" "/lib/profiling-lib.sh"
+
+    # shellcheck disable=SC2154
+    cat << EOF >> "${initdir}/etc/profiling.conf"
+export trace_term=${trace_term}
+export trace_baud=${trace_baud}
+EOF
+
+    # optionally enable early Dracut profiling
+    if [ -n "${dracut_early_tracing}" ]; then
+      inst_hook cmdline 00 "${moddir}/profiling/profiling-lib.sh"
+    fi
+  else
+    # Install the default profiling library that is a simple `return 0`
+    inst_simple "${moddir}/profiling/profiling-lib-noop.sh" "/lib/profiling-lib.sh"
```

Comment:
To avoid keeping some no-op file around, should this just be
```sh
echo "return 0" > "${whatever-the-installation-root-variable-is}/lib/profiling-lib.sh"
```

Reply:
Done - I originally had done this but then opted to use a file-on-disk for some unknown reason. Restored to this approach.